### ISSUE TITLE
Right hand side driving and parking attribute

### ISSUE THREAD

**Issue by asheesh1202** [valhalla/valhalla#2283]
While extracting the features from baldr, RHS driving and parking attributes are set as false for all the nodes. Is this expected?

**Comment by asheesh1202** [comment]
Any Idea why is it like this??

**Comment by dgearhart** [comment]
@gknisely is this because of missing admin db?

**Comment by asheesh1202** [comment]
admin db is available. but how to read information from the db.

**Comment by gknisely** [comment]
Yes.  @asheesh1202 you need to build the admin db via `build/valhalla_build_admins`  When it is complete, make sure you update your config to point to it.  Also, you should build the timezone db as well and point to it.  Build it via `/scipts/valhalla_build_timezones`

**Comment by asheesh1202** [comment]
Ok. 
But then is there any method that does reading of these properties from the Databases?

**Comment by asheesh1202** [comment]
@gknisely is this required to be done when we are building tiles? - the adding of  admin and time zone paths in config file.
Do I require to do anything after the this.

**Comment by gknisely** [comment]
@asheesh1202 you need to rebuild the tiles

**Comment by asheesh1202** [comment]
I tried  it, didn't work for me. :(

**Comment by gknisely** [comment]
@asheesh1202 the admin db contains[driving side information](https://github.com/valhalla/valhalla/blob/master/lua/admin.lua) and when we build the tiles we do basically a pip and determine what country you are in for each node.  Sound like something either failed or you are not configured properly.

**Comment by asheesh1202** [comment]
And is this is related to parking attribute also??
